.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,28 @@
+name: Feature request
+description: Ask for a feature to be implemented and included in the next release
+labels: feature request
+title: "[REQUEST] "
+body:
+
+    - type: markdown
+      attributes:
+        value: |
+            Please make sure to [search issues](https://github.com/mawww/kakoune/issues?q=is%3Aissue) before creating a new one, to avoid duplicates and centralise conversations.
```

Comment:
Yeah, there are so many duplicates. Also some open issues are long fixed.
We should close them all. That will make it easier for users to find relevant
issues.